{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1204, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1204, "_consider_importhook"]}
{"pytest_version": "7.4.4", "$report_type": "SessionStart"}
{"nodeid": "tests49_82_4.py::test_binary_of_two", "location": ["tests49_82_4.py", 11, "test_binary_of_two"], "keywords": {"test_binary_of_two": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00015723100113973487, "start": 1729822606.417328, "stop": 1729822606.4174855, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw1>"}
{"nodeid": "tests49_82_4.py::test_binary_of_zero", "location": ["tests49_82_4.py", 3, "test_binary_of_zero"], "keywords": {"test_binary_of_zero": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00020862100063823164, "start": 1729822606.4175365, "stop": 1729822606.417746, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw0>"}
{"nodeid": "tests49_82_4.py::test_binary_of_one", "location": ["tests49_82_4.py", 7, "test_binary_of_one"], "keywords": {"test_binary_of_one": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00017356900025333744, "start": 1729822606.4183795, "stop": 1729822606.4185538, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw2", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw2>"}
{"nodeid": "tests49_82_4.py::test_binary_of_one", "location": ["tests49_82_4.py", 7, "test_binary_of_one"], "keywords": {"test_binary_of_one": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_one():", ">       assert not if_decimal_is_divisible('1')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 9, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def get_next_fibonacci(current):", "        next = current + fibonacci(current)", ">       while next < 99:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["current", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_one():", ">       assert not if_decimal_is_divisible('1')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 9, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def get_next_fibonacci(current):", "        next = current + fibonacci(current)", ">       while next < 99:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["current", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (129568081970752) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 6.995471861000624, "start": 1729822606.4231088, "stop": 1729822613.418582, "extras": [], "$report_type": "TestReport", "item_index": 1, "worker_id": "gw2", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw2>"}
{"nodeid": "tests49_82_4.py::test_binary_of_one", "location": ["tests49_82_4.py", 7, "test_binary_of_one"], "keywords": {"test_binary_of_one": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (129568081970752) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.00016821600002003834, "start": 1729822613.432735, "stop": 1729822613.4329042, "$report_type": "TestReport", "item_index": 1, "worker_id": "gw2", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw2>"}
{"nodeid": "tests49_82_4.py::test_binary_of_two", "location": ["tests49_82_4.py", 11, "test_binary_of_two"], "keywords": {"test_binary_of_two": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_two():", "        if 99 == 4:", "            assert if_decimal_is_divisible('10')", "        else:", ">           assert not if_decimal_is_divisible('10')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 16, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def get_next_fibonacci(current):", "        next = current + fibonacci(current)", ">       while next < 99:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["current", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_two():", "        if 99 == 4:", "            assert if_decimal_is_divisible('10')", "        else:", ">           assert not if_decimal_is_divisible('10')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 16, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def get_next_fibonacci(current):", "        next = current + fibonacci(current)", ">       while next < 99:", "E       Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["current", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 10, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139093417657920) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 7.0000774760010245, "start": 1729822606.4177945, "stop": 1729822613.4178731, "extras": [], "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw1>"}
{"nodeid": "tests49_82_4.py::test_binary_of_two", "location": ["tests49_82_4.py", 11, "test_binary_of_two"], "keywords": {"test_binary_of_two": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139093417657920) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.0006292199996096315, "start": 1729822613.436322, "stop": 1729822613.4369524, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw1>"}
{"nodeid": "tests49_82_4.py::test_binary_of_zero", "location": ["tests49_82_4.py", 3, "test_binary_of_zero"], "keywords": {"test_binary_of_zero": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_zero():", ">       assert if_decimal_is_divisible('0')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 5, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    next = next + fibonacci(next)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 11, "message": "in get_next_fibonacci"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   def fibonacci(n):", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["n", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_binary_of_zero():", ">       assert if_decimal_is_divisible('0')"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/tests49_82_4.py", "lineno": 5, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    next_fibonacci = get_next_fibonacci(0)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 19, "message": "in if_decimal_is_divisible"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    next = next + fibonacci(next)"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 11, "message": "in get_next_fibonacci"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   def fibonacci(n):", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [["n", "0"]]}, "reprlocals": null, "reprfileloc": {"path": "Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 3, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/ASE2024_GitHub/Turbulence-Benchmark-main/CodeGemma7B_T_0/Q49/codegemma_results_4/Folder_82/generated_answer.py", "lineno": 3, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (137848070080064) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 7.003201727999112, "start": 1729822606.4180603, "stop": 1729822613.4212632, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw0>"}
{"nodeid": "tests49_82_4.py::test_binary_of_zero", "location": ["tests49_82_4.py", 3, "test_binary_of_zero"], "keywords": {"test_binary_of_zero": 1, "tests49_82_4.py": 1, "Folder_82": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [["Captured stdout call", "~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (137848070080064) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 411, in _perform_spawn\n    reply.run()\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 341, in run\n    self._result = func(*args, **kwargs)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 1160, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 567, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/execnet/gateway_base.py\", line 534, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 0.000113303000034648, "start": 1729822613.439232, "stop": 1729822613.4393458, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "35f5a9a723ba433d83728d3986e68fae", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
